docs/source/quickstart/setup.mdx

```diff
@@ -135,7 +135,7 @@ type SubmitReviewResponse {
 
 ## 1. Install the Rover CLI
```

Comment:
Add a note for folks to check their Rover version after running the `curl` command. :)